[DOCS] add setup guidance for using the Python api with GX Cloud (#9168)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/set_up_gx_cloud.md

    ```      
    Replace `user_access_token` and `organization_id` with the values you copied previously, and `snowflake_password` with your own value.

> 3. Optional. To use GX Cloud in Python scripts: save your `GX_CLOUD_ACCESS_TOKEN` and `GX_CLOUD_ORGANIZATION_ID` as environment variables outside of the Docker Engine by entering `export ENV_VAR_NAME=env_var_value` in the terminal or adding the commands to your `~/.bashrc` or `~/.zshrc` file. For example:

Reviewer: ```suggestion
3. Optional. To save your `GX_CLOUD_ACCESS_TOKEN` and `GX_CLOUD_ORGANIZATION_ID` environment variables outside of Docker, enter `export ENV_VAR_NAME=env_var_value` in the terminal or add the commands to your `~/.bashrc` or `~/.zshrc` file. For example:
```
This seems an odd addition. Why inform users that they don't need to use Docker when this is the preferred methodology? Could this cause confusion?

Author: This is solely included so that people can access their GX Cloud account while automating processes through Python scripting.  The mention of using GX Cloud in Python scripts was included to indicate the use case where someone would want to save their environment variables outside of Docker. It could be considered an "advanced" use case, but it still requires a setup step that someone who intends to use it would expect to find here.  

Once this has been done the user would be able to switch over to the OSS docs for further guidance, but their scripts would now access their GX Cloud account through a GX Cloud Data Context that was initialized in the Python script, instead of a File Data Context stored somewhere outside GX Cloud.
